docs/src/main/paradox/java/http/migration-guide/migration-within-10.0.x-Api-May-Change.md

```diff
@@ -0,0 +1,14 @@
+# Migration Guide within Akka HTTP 10.0.x
+
+## General Notes
+Akka HTTP is binary backwards compatible within for all version within the 10.0.x range. However, there are a set of APIs
+that are marked with the special annotation `@ApiMayChange` where this binary backwards compatibility is not guaranteed.
```

Comment:
are marked with the special annotation `@ApiMayChange` where this binary backwards compatibility is not guaranteed.

=>

are marked with the special annotation `@ApiMayChange` which are exempt from this rule, in order to allow them to be evolved more freely until stabilising them, by removing this annotation.

?